app/controllers/press_catalog_controller.rb

```diff
@@ -49,6 +49,18 @@ def load_press
       render file: Rails.root.join('public', '404.html'), status: :not_found, layout: false
     end
 
+    def search_ongoing
+      @search_ongoing = false
+
+      params.each do |key, _value|
+        non_search_keys = ['action', 'authenticity_token', 'controller', 'locale', 'page', 'per_page', 'press', 'sort', 'utf8', 'view']
```

Comment:
Brittle logic but I can't think of an alternative.

Reply:
Yes, it's totally possible that other params will need to be added after running/testing this for a while.
We could reverse the logic and actually store all the Solr field names for facets instead? I can go that way if you think it's worth it. Later could add human-readable names in such a list as well, to use in a very elaborate message for https://tools.lib.umich.edu/jira/browse/HELIO-3478
I was planning on avoiding both of those things because it seems like adding maintenance nightmares on top of the feature creep. Dunno.

Reply:
Yeah, I was thinking the search_keys would be a much bigger and uglier set hence I can stomach the brittle in this context.

Reply:
OK resolved in parking lot. Good enough for now.